src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -96,6 +96,10 @@ const HeaderSubTitle = styled.div`
   line-height: 28px;
   font-family: 'Poppins', sans-serif;
 
+  @media (max-width: 768px){
+  justify-content: left ; 
+  }
+
```

Comment:
Please check indentation
```suggestion
  @media (max-width: 768px){
      justify-content: left; 
  }

```